src/screens/LoginScreen.js

```diff
@@ -130,7 +130,7 @@ const styles = StyleSheet.create({
         padding: 20,
         paddingLeft: 40,
         paddingRight: 40,
-        backgroundColor: headerWhiteTransparent
+        // backgroundColor: headerWhiteTransparent,
```

Comment:
@amgoodman5 I cleaned up a bug that we had that was hiding where if we make two imports from the same file (we did this a lot with `import {...} from '../styles'` and then a few lines down `import {...} from '../styles/index'` the app would crash with the new SDK. What previous versions of the SDK were hiding, however, was that the second import wasn't actually doing anything. In this case, the import of `headerWhiteTransparent` for the `LoginScreen` wasn't actually coming through. I left it commented in case you want to keep it and play with the styling for it.